### PR TITLE
DS-2230/rename

### PR DESCRIPTION
**Pull request by gsganden**
# Pull Request Process

Contributors from outside ShopRunner should feel free to submit a PR without having completed all of the items below. See [CONTRIBUTING.md](https://github.com/ShopRunner/wildebeest/blob/master/CONTRIBUTING.md) for additional information.

You can run `./.ci/test.sh` locally to check for style issues, run tests, rebuild docs, etc. before submitting changes.

## Description

Changing the name of the repo in light of J.K. Rowling's comments on transgender issues. Moving from Jenkins to GitHub Actions at the same time to avoid having to make two sets of CI/CD changes.

### General

- [x] Pull request [uses keywords](https://help.github.com/en/articles/closing-issues-using-keywords) to close relevant [issues](https://github.com/ShopRunner/wildebeest/issues).
- [x] Pull request includes unit tests for any bug fixes and new functionality.
- [x] Docs have been updated as needed. (`sphinx-build docs docs/_html` rebuilds the docs. Open `docs/_html/index.html` to check them.)

### ShopRunner Contributors

The maintainer will complete the following steps for contributions from outside ShopRunner.

- [x] `CHANGELOG.md` has been updated.
- [x] `_version.py` has been updated.
- [x] Version number has been updated in `docs/conf.py`.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,22 +1,25 @@
...
+# Pull Request Process
 
-**General Pull Request Checklist**
+Contributors from outside ShopRunner should feel free to submit a PR without having completed all of the items below. See [CONTRIBUTING.md](https://github.com/ShopRunner/wildebeest/blob/master/CONTRIBUTING.md) for additional information.
 
- - [ ] Pull request has been made against the `wip` branch (or from `wip` into `master`).
- - [ ] Pull request includes a description of the change and the reason behind it.
- - [ ] Pull request [uses keywords](https://help.github.com/en/articles/closing-issues-using-keywords) to close relevant [issues](https://github.com/ShopRunner/creevey/issues).
- - [ ] Pull request includes unit tests for any bug fixes and new functionality.
- - [ ] `./.ci/test.sh` passes locally.
```
Comment (gsganden) [inline .github/pull_request_template.md:9]: IMO you shouldn't have to certify that you've run checks locally; that's what CI/CD is for.

File: requirements-dev.in [hunk 2]
```diff
@@ -4,8 +4,12 @@ black
 flake8
 flake8-docstrings
 flake8-import-order
+isort
+git+https://github.com/crossnox/m2r@dev#egg=m2r
```
Comment (parsing-science) [inline requirements-dev.in:8]: Why did we have to switch to using a library from github?

File: tests/conftest.py [hunk 3]
```diff
@@ -1,8 +1,8 @@
-from functools import partial
 import sys
+from functools import partial
```
Comment (parsing-science) [inline tests/conftest.py:2]: Shouldn't this line come from before `sys`?

Comment (gsganden) [reply tests/conftest.py:1]: Fixed

File: requirements-dev.in (hunk 2 above)
Comment (gsganden) [reply requirements-dev.in:8]: See https://github.com/sphinx-doc/sphinx/issues/7420

Comment (gsganden) [reply requirements-dev.in]: I added a comment to requirements-dev.in

**Pull request merged**